installing/install_config/customizations.adoc

```diff
@@ -97,7 +97,7 @@ to control which example imagestreams and templates are installed on the cluster
 == Additional configuration resources
 
 These configuration resources represent a single instance of a particular component, in some cases multiple
-instances can be requested by creating multiple instances of the resource.  In other cases only a specific 
+instances can be requested by creating multiple instances of the resource.  In other cases only a specific
```

Comment:
Also not in scope but could use a period and commas inserted: 
s/component, in some cases multiple/component. In some cases, multiple/
s/In other cases only/In other cases, only